docs/source/index.rst

```diff
@@ -75,6 +77,12 @@ or install it using your system package manager, e.g.::
 
     sudo apt-get install python3-pyqt5
 
+or::
+
+    sudo apt-get install python-pyqt
+    sudo apt-get install python3-pyqt
+
+if PyQt4 is desired.
```

Comment:
Maybe in the paragraph above, we should expand "PyQt or PySide" to say "PyQt5, PyQt4 or PySide".

I don't think there's any great harm in putting these in, but it seems like superfluous information. There's a lot of potential combinations of packaging system, Python version and Qt binding, and I don't want to end up having to make a great big list of possible commands. I'd rather give one example command and let people work out how to adapt it.